src/transformStateWithClones.js

```diff
@@ -66,6 +66,39 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const stateClone = { ...state };
+  const states = [];
+
+  for (const transform of transforms) {
```

Comment:
also you could do destructuring of this variable for easier use 
```suggestion
  for (const { operation, properties } of transforms) {
```